modules/openapi-generator/src/main/resources/python/api.mustache

```diff
@@ -6,19 +6,15 @@ import io
 import warnings
 
 from pydantic import validate_call, Field, StrictFloat, StrictStr, StrictInt
-from typing import Dict, List, Optional, Tuple, Union, Any
-
-try:
-    from typing import Annotated
-except ImportError:
-    from typing_extensions import Annotated
+from typing import Any, Dict, List, Optional, Tuple, Union
+from typing_extensions import Annotated
 
 {{#imports}}
 {{import}}
 {{/imports}}
 
 from {{packageName}}.api_client import ApiClient
-from {{packageName}}.api_response import ApiResponse
+from {{packageName}}.api_response import ApiResponse as _ApiResponse
```

Comment:
Why is the alias needed here?

Reply:
@wing328 do you have any suggestion on how to prevent conflicts for the internal classes/types generated by the generator itself and what the OpenAPI spec needs?

The problem here is that:

* The Pet Store API generates [an `ApiResponse` object](https://github.com/OpenAPITools/openapi-generator/blob/8b5b5a74c333b809c5a651366656257ec8a6fef3/samples/openapi3/client/petstore/python/petstore_api/models/api_response.py#L28)
* It conflicts with [the internal `ApiResponse`](https://github.com/OpenAPITools/openapi-generator/blob/8b5b5a74c333b809c5a651366656257ec8a6fef3/samples/openapi3/client/petstore/python/petstore_api/api_response.py#L9) (the generic response object holding the result of HTTP calls before being serialized to the actual response objects)

Reply:
Ah no, nevermind, I can actually just override the generated ApiResponse model :facepalm: